### PR TITLE
Adding the test case for fixed-array chunked dataset and VDS

### PR DESCRIPTION
**Pull request by raylu-hdf**
My first PR to VFD SWMR.  Please let me know if I did anything ridiculous.

### CODE REVIEW

File: test/vfd_swmr_bigset_writer.c [hunk 1]
```diff
@@ -370,11 +375,26 @@ state_init(state_t *s, int argc, char **argv)
     s->chunk_dims[0] = s->rows;
     s->chunk_dims[1] = s->cols;
     s->one_dee_max_dims[0] = s->rows;
-    s->one_dee_max_dims[1] = H5S_UNLIMITED;
+    if(s->fixed_array) {
```
Comment (vchoi-hdfgroup) [inline test/vfd_swmr_bigset_writer.c:381]: Write description/comments for the overall changes you make to vfd_swmr_bigset_writer.c for fixed array e.g. why the handling is different from current coding.

Comment (raylu-hdf) [reply test/vfd_swmr_bigset_writer.c:381]: I added some comments.  Hopefully, they are sensible.

File: test/vfd_swmr_bigset_writer.c [hunk 2]
```diff
@@ -207,11 +210,12 @@ newmat(unsigned rows, unsigned cols)
 static void
 usage(const char *progname)
 {
-	fprintf(stderr, "usage: %s [-S] [-W] [-a steps] [-b] [-c cols]\n"
+	fprintf(stderr, "usage: %s [-F] [-M] [-S] [-V] [-W] [-a steps] [-b] [-c cols]\n"
                 "    [-d dims]\n"
                 "    [-n iterations] [-r rows] [-s datasets]\n"
                 "    [-u milliseconds]\n"
 		"\n"
+                "-F:                   fix maximal dimensions for the chunked datasets\n"
```
Comment (vchoi-hdfgroup) [inline test/vfd_swmr_bigset_writer.c]: Minor: "Fixed maximal dimensions..."

**Review by vchoi-hdfgroup — APPROVED** [review]

File: test/vfd_swmr_bigset_writer.c (hunk 2 above)
Comment (raylu-hdf) [reply test/vfd_swmr_bigset_writer.c]: Corrected.

**Pull request merged**